frontend/src/bundles/common/helpers/chart-helper.ts

```diff
@@ -16,4 +25,70 @@ const getData = (data: DataObject[]): number[] => {
     return data.map(({ value }) => value);
 };
 
-export { convertDate, getData, getLabels };
+const createLineChartOptions = (
+    tooltipDisplay: boolean,
+): ChartOptions<'line'> => ({
+    responsive: true,
+    plugins: {
+        legend: {
+            display: false,
+        },
+        tooltip: {
+            enabled: tooltipDisplay,
+        },
+    },
+    layout: {
+        padding: PADDING,
+    },
+    scales: {
+        x: {
+            grid: {
+                color: GRID_COLOR,
+                lineWidth: BORDER_WIDTH,
+                tickWidth: 0,
+            },
+            type: 'time',
+            time: {
+                unit: 'day',
+            },
+            ticks: {
+                font: {
+                    size: FONT_SIZE,
+                },
+                autoSkip: true,
+                maxRotation: 25,
+                minRotation: 25,
+                align: 'inner',
+                color: TEXT_COLOR,
+                maxTicksLimit: 8,
+                callback: function (value): string {
+                    return convertDate(value);
+                },
+            },
+        },
+        y: {
+            grid: {
+                color: GRID_COLOR,
+                lineWidth: BORDER_WIDTH,
+                tickWidth: 0,
+            },
+            min: -200,
+            ticks: {
+                color: TEXT_COLOR,
+                maxTicksLimit: 6,
+                callback: function (value, index, ticks): string {
+                    const newValue =
+                        index == ticks.length - 1
+                            ? ticks[ticks.length - 2].value * 2
```

Comment:
Create a variable for this calculation

Reply:
Resolved in [1e7b75e](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/160/commits/1e7b75e2bf5b6a4c7a434b1fa8791730554ddf35)